models/cam-fv/readme.rst

```diff
@@ -165,16 +171,16 @@ which contain the types of observations in the table below
 ("T" = temperature, "U" = zonal wind, "V" = meridional wind,
 "Q" = specific humidity, "refractivity" = the bending of light by density variations).
 These files are available on NSF NCAR's glade file system:
-/glade/p/cisl/dares/Observations/NCEP+ACARS+GPS+AIRS/Thinned_x9x10.
+/glade/campaign/cisl/fs1_p_relocation/dares/Observations/NCEP+ACARS+GPS+AIRS/Thinned_x9x10.
 Versions of these files, which also have the results of the reanalysis in them,
-are available from the RDA ds345.0 linked above.
+are available from the RDA d345000 linked above.
 
 NCEP
    NCEP's PREPBUFR files (prepqm) in NSF NCAR's Research Data Archive:
-   (https://rda.ucar.edu/datasets/ds090.0/)
+   `newer <NCEP_obs_>`_ or `older <NCEP+NCAR_obs_>`_
 
 COSMIC
-   This site (http://www.cosmic.ucar.edu/) provides atmospheric refractivity 
+   `This site <http://www.cosmic.ucar.edu/>`_ provides atmospheric refractivity 
```

Comment:
name the site rather 'COSMIC', or  www.cosmic.ucar.edu rather than 'this site'

Reply:
Good suggestion.  I was just making the minimal changes.